v2.10.2/harbor-core/rockcraft.yaml

```diff
@@ -0,0 +1,189 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+# Rockcraft definition for Harbor core image:
+# docker.io/goharbor/harbor-core:v2.10.2
+
+name: harbor-core
+summary: Rock containing Harbor Core components.
+description: |
+  Packages the core service of Harbor.
+license: Apache-2.0
+
+version: "v2.10.2"
+
+# NOTE(aznashwan): the base for the core image is VMware's Photon,
+# but rockcraft only currently supports bare/ubuntu-based bases.
+base: ubuntu@22.04
+build-base: ubuntu@22.04
+platforms:
+  amd64:
+  # arm64:
```

Comment:
We do, though I had disarmed building on ARM (pun not intended) since we can't configure ARM runners on our forks and forgot to add a comment about it...

I've re-instated them now and will use a different branch on my fork for testing the workflows.